notebooks/254-llm-chatbot/254-llm-chatbot.ipynb

```diff
@@ -83,7 +83,7 @@
     "\"git+https://github.com/openvinotoolkit/nncf.git\"\\\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    "openvino&gt;=2024.0.0"\
</pre><p>what hte reason to use 2024.0 version only?</p><p>I know about limitations to run llama and gemma with transformers&gt; 4.38  for this version</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1824/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Because the PR to fix GPU issue is not merged yet, CVS-133452

Reply:
done